server.js

```diff
@@ -1,35 +1,171 @@
 import express from "express";
 import cors from "cors";
 import mongoose from "mongoose";
+//import dotenv from "dotenv"
+import listEndpoints from "express-list-endpoints";
+import topMusicData from "./data/top-music.json";
+
+//to do: envfile for sensitive data (enviromental variables, external file)
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
 
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
 mongoose.Promise = Promise;
 
+//property keys saved as the model, User, Song
+//mongoose user model - the structure
+// const User = mongoose.model("User", {
+// name: String,
+// age: Number,
+// deceased: Boolean
+// }); 
+
+//mongose song model - the structure
+//object property keys name, age and number
+const Song = mongoose.model("Song", {
+  id: Number,
+  trackName: String,
+  artistName: String,
+  genre: String,
+  bpm: Number,
+  energy: Number,
+  danceability: Number,
+  loudness: Number,
+  liveness: Number,
+  valence: Number,
+  length: Number,
+  acousticness: Number,
+  speechiness: Number,
+  popularity: Number
+});
+
+// choose when we want to delete. when if(true) and will always happen
+// put and delete entries in database
+  if(process.env.RESET_DB) {
+  const resetDataBase = async () => {
+    await Song.deleteMany();
+    topMusicData.forEach(singleSong => {
+      const newSong = new Song(singleSong);
+      newSong.save();
+    })
+  }
+  resetDataBase(); //DO NOT EVER DO THIS IN PRODUCTION ENVIROMENT!
```

Comment:
Good idea with commenting the part that was educational but we should not use in the production enviroment